Makefile.am

```diff
@@ -183,6 +183,7 @@ if ENABLE_CCATOK
 	if test -d $(DESTDIR)/$(libdir)/opencryptoki/stdll; then \
 		cd $(DESTDIR)/$(libdir)/opencryptoki/stdll && \
 		rm -f PKCS11_CCA.so; fi
+	rm -f $(DESTDIR)$(sysconfdir)/opencryptoki/ccatok.conf 
```

Comment:
Are we then removing /etc/opencryptoki/ccatok.conf ?
What if the user customized the conf file? I see we are already removing the ep11tok.conf file. So yes, let's make this consistent, but I'm not sure what's right ;-)

Reply:
Yes, we are removing all files that we installed with `make install`. 

Please note that `make uninstall` is not the same as an RPM uninstall. An RPM uninstall usually takes care about modified config files during install (don't overwrite if modified), and uninstall (keep if modified). 

Please also note that as part of `make distcheck` it not only builds the package, but also installs it via `make install` into a temporary location, and then uninstalls it via `make uninstall`. It then checks if all files there were installed got removed during uninstall. If not, then `make distcheck` fails. So this tells me that `make uninstall` must remove all files that are installed with `make install`.